### PR TITLE
Added search by name and some scenescraper fixes.

### PR DESCRIPTION
**Pull request by ryosaeba75**

### CODE REVIEW

File: scrapers/DorcelVision.yml [hunk 1]
```diff
@@ -10,34 +10,65 @@ movieByURL:
...
+            - regex: (.*)
+              with: https://www.dorcelvision.com$1
+              
+      Image: 
+        selector: $card/a[@class="movies rolloverv2"]/img[last()]/@data-src
+        postProcess: 
+          - replace:
+              - regex: (.*)
+                with: https://www.dorcelvision.com$1
+      Details: $card/details/summary
```
Comment (Maista6969) [inline scrapers/DorcelVision.yml:42]: Some of these have longer descriptions, try this:
```suggestion
      Details: $card/details//p[last()]
```

File: scrapers/DorcelVision.yml [hunk 2]
```diff
@@ -10,34 +10,65 @@ movieByURL:
...
+
```
Comment (Maista6969) [inline scrapers/DorcelVision.yml:43]: Let's grab the performers while we're at it 👍 
```suggestion
      Performers:
        Name:
          selector: $card/details//p[@class="persons"]
          postProcess:
            - replace:
                - regex: ^With:\s+
                  with:
                - regex: \s*,\s*
                  with: __SEPARATOR__
          split: __SEPARATOR__
```

**Review by Maista6969 — APPROVED** [review]
Looks great, I only have two suggestions but I can merge this right away and I can fix 'em if you'd like 👍

File: scrapers/DorcelVision.yml (hunk 2 above)
Comment (ryosaeba75) [reply scrapers/DorcelVision.yml:43]: I couldn't manage to make this work when I tried, it returns always the same list of performers for every result, even ones where is no performers listed.

Comment (Maista6969) [reply scrapers/DorcelVision.yml:43]: I apologize, I thought I tested this but upon retrying it now I can see the exact issue you're describing: disregard this comment 😅

**Pull request merged**